selendroid-standalone/src/main/java/io/selendroid/standalone/SelendroidConfiguration.java

```diff
@@ -91,15 +91,40 @@
   private boolean keepAdbAlive = false;
 
   @Parameter(names = "-maxSession", description = "Maximum number of sessions that a grid hub can assign at a time.")
-  private int maxSession = 5;
+  private int maxSession = 0;
```

Comment:
ah, i see it defaults to the devices available... maybe might want to change the description?